src/core/display/Container.js

```diff
@@ -459,14 +467,22 @@ export default class Container extends DisplayObject
         }
 
         // add this object to the batch, only rendered if it has a texture.
-        this._renderWebGL(renderer);
```

Comment:
Since it's exactly the same code as above, then maybe separate into private function that could be reused in both places?

Reply:
sure. I will change this